components/filecoin/rs/src/signature.rs

```diff
@@ -0,0 +1,196 @@
+use crate::message::MessageAPI;
+use blake2b_simd::Params;
+use bls_signatures::Serialize;
+use core::{array::TryFromSliceError, num::ParseIntError};
+use fvm_ipld_encoding::Cbor;
+use fvm_shared::address::Protocol;
+use fvm_shared::message::Message as UnsignedMessage;
+use libsecp256k1::util::{SECRET_KEY_SIZE, SIGNATURE_SIZE};
+use libsecp256k1::{sign, Message, RecoveryId};
```

Comment:
Since you use the namespaced type defining `SecpSignature` below (which I like!) you don't need to import the unprefixed `RecoveryId` here.

Reply:
done